docs/.vuepress/plugins.js

```diff
@@ -368,6 +368,14 @@ module.exports.officialPlugins = [
     description: "Adds support to compile Vyper smart contracts",
     tags: ["Vyper", "Compiler"],
   },
+    {
+    name: "@ControlCplusControlV/hardhat-Yul",
```

Comment:
have you published it to npm? i can't find the package

Reply:
Oh no I have not, I will do that now!

Reply:
Published now https://www.npmjs.com/package/@controlcpluscontrolv/hardhat-yul